packages/devextreme/testing/tests/DevExpress.ui.widgets/chat.tests.js

```diff
@@ -134,6 +146,44 @@ QUnit.module('Chat initialization', moduleConfig, () => {
 
         assert.strictEqual(invalidateStub.callCount, 1);
     });
+
+    QUnit.test('Last message should be into view', function(assert) {
```

Comment:
There is no tests that check scroll position after initialization.

We can declare the reinit() utility method to simplify routine actions as follows:
https://github.com/marker-dao/DevExtreme/blob/24_2_chat_scrolling/packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/dateRangeBox.tests.js#L74

Reply:
[5f471b926b7031dac4c622c538f731e7447e4126]